.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# application code
+/securedrop/ @redshiftzero
+/securedrop/ @heartsucker
```

Comment:
Let's also add @dachary as a code owner for `securedrop/*`

Reply:
👍